.github/workflows/ci.yml

```diff
@@ -7,7 +7,8 @@ on:
       - '!master'
 
 env:
-  BUILDER_VERSION: v0.5.9
+  BUILDER_VERSION: android-cross
+  BUILDER_SOURCE: channels
```

Comment:
should this get merged?

Reply:
Nope, depends on the merge/release of builder